azure-pipelines.yml

```diff
@@ -361,6 +372,54 @@ jobs:
 
     displayName: 'Test pip Install'
 
+- job: UnitTest
+  displayName: Unit Test for Core and Telemetry
+  timeoutInMinutes: 10
+
+  pool:
+    vmImage: 'ubuntu-16.04'
```

Comment:
why ubuntu only?

Reply:
I think we should add it in Windows too.

Reply:
Oh, sorry, currently, there is no script supported in Windows platform. All test script are suffix with *.sh. 

So I create another issue to fix that https://github.com/Azure/azure-cli/issues/11450 in S163